new/LIT6965Egziabfatare.xml

```diff
@@ -32,13 +32,15 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
         <profileDesc>
             <creation/>
             <abstract>
-                <p>This poem is referred to as qǝne of the type <foreign xml:lang="gez">kǝbr yǝʾǝti</foreign>,
+                <p>This poem is referred to as a <term key="Qene"/> of the type <foreign xml:lang="gez">kǝbr yǝʾǝti</foreign>,
                     in <ref type="mss" corresp="ESdd037"/>, and attributed to <persName ref="PRS14409Yohannes"/>.</p>
             </abstract>
             <textClass>
                 <keywords>
                     <term key="ChristianLiterature"/>
                     <term key="Prayers"/>
+                    <term key="Qene"/>
+                    <term key="EtanaMogar">ʿǝṭāna mogar</term>
```

Comment:
not EtanaMogar, KebrYeeti? please provide suitable keywords, also in the text

Reply:
done